admin-dev/themes/new-theme/js/pages/product/components/combination-modal/CombinationModal.vue

```diff
@@ -198,6 +196,10 @@
       };
     },
     props: {
+      paginatedCombinationsService: {
```

Comment:
Oh nice it's better to inject it via props indeed. Actually, I'm dumb it's mandatory if we want filters to be global even for the modal 😅

I'm not sure but it seems productId is now useless and could be removed